package.json

```diff
@@ -16,7 +16,7 @@
     "index.js"
   ],
   "peerDependencies": {
-    "eslint": ">=1.6.0 <6.0.0",
+    "eslint": ">=1.6.0 <7.0.0",
```

Comment:
Not sure this loader works fine with `eslint@1`/`eslint@2`/`eslint@3`/`eslint@4`, but it is out of scope this PR

Reply:
Checked on `eslint@^4`, `eslint@^5`, `eslint@^6`. Tests is passing.